### DISCUSSION TITLE
All open Issues are confusing

### DISCUSSION THREAD

**Discussion by l4ejyetrgf** [Chatterino/chatterino2#4080] [Q&A] [answered]
Some Issues exist three or more times and are not linked or closed.
Some Issues are from old versions, and the author didn't reply back.
Handful pulls are ready for review and will not be merged.
So many issues and Pulls have been open for many years now, and maybe someone can decide which can be closed, 
which can easily be fixed, and 
which is good for implementation. 
Please clean up the repo, it helps a lot and makes creating new issues or pulls much simpler.

**Comment by pajlada** [comment, marked as answer]
Thanks for your feedback - this is mostly a time constraint issue. Maintaining medium size open source projects like this can take up a lot of free time so I try to time-gate it to a few hours per day. This leads to issues you detailed (issues left on read, PRs not being merged in a timely manner) but there's not much to do about it from my perspective so if feature requests/bug reports keep coming in at the velocity they are, things will keep piling up and that's alright :slightly_smiling_face:

**Comment by Felanbird** [comment]
> Some Issues exist three or more times and are not linked or closed.

Where are some examples?

> Some Issues are from old versions, and the author didn't reply back.

Link some examples.

> Handful pulls are ready for review and will not be merged.

"will not be merged" - I like that your assumption is we just wont merge them, and that there's no other reason(s) preventing said merging.

> which can easily be fixed, 

Fix them then, you can PR to open PRs

> Please clean up the repo, it helps a lot and makes creating new issues or pulls much simpler.

No it doesn't, it makes it so once an issue is closed, people can say "We already closed this issue it wont be added", for stuff that could and likely should be implemented.

**Reply by l4ejyetrgf** [reply]
Some Issues sound like the same problem #2559 #2516 / #2056 #1228 / #1540 #2728 Maybe i am dumb

#2392

#3884

I am technically not experienced.
#2955

**Reply by Felanbird** [reply]
https://github.com/Chatterino/chatterino2/pull/3884 has no update because goldbattle is not active, while he may have bumped all of his PRs, we reviewed 2 PRs from them, and they are  awaiting changes for them to apply, so there's no reason to rush into reviewing a 3rd one that will stay stale.

https://github.com/Chatterino/chatterino2/issues/1540 suggests a less confusing* setup, while https://github.com/Chatterino/chatterino2/issues/2728 wants a more advanced setup, which would likely cause more confusion


Both https://github.com/Chatterino/chatterino2/issues/2559 & https://github.com/Chatterino/chatterino2/issues/2516 / https://github.com/Chatterino/chatterino2/issues/2056 & https://github.com/Chatterino/chatterino2/issues/1228 are Windows vs Linux reports, and due to the differences between the operating systems, it's very possible the way they need to be fixed is different, so I don't feel confident closing them.

https://github.com/Chatterino/chatterino2/issues/2392 is fair, and I will close this one, but this was way more effort than either of us should have had to put in, to close 1 issue.

**Comment by leon-richardt** [comment]
Duplicate issues are definitely something that can "clog up" a repository. Especially in our case, with 352 open issues at the time of writing, it can be easy to miss an existing issue. Sometimes, however, issues that read the same to an "outsider" have subtle difference that justify their respective existences.

Nonetheless, if you feel like you have found two identical issues, please leave a comment on the newer issue with a link to the older one. That way, we can gradually make progress in terms of cleaning them up.
